pkg/crc/machine/bundle/metadata_test.go

```diff
@@ -209,5 +209,9 @@ func TestVerifiedHash(t *testing.T) {
 func testDataURI(t *testing.T, sha256sum string) string {
 	absPath, err := filepath.Abs(filepath.Join("testdata", sha256sum))
 	require.NoError(t, err)
+	// when its a windows path with drive letter
+	if volume := filepath.VolumeName(absPath); volume != "" {
+		return fmt.Sprintf("file://%s/%s", volume[:1], absPath[3:])
+	}
```

Comment:
`file:///D:/a/crc/crc/pkg/crc/machine/bundle..` is apparently acceptable, see https://github.com/golang/go/issues/6027 , so maybe calling https://pkg.go.dev/path/filepath#ToSlash would be enough to get a functional URI?

Reply:
It didn't work, even after passing it to `filepath.ToSlash` the resulting URI is `D:\D:\a\crc\crc\pkg\crc\machine\bundle\testdata\sha256sum_correct_4.13.0.txt.sig`

`filepath.Abs` already returns `D:\a\crc\crc\pkg\crc\machine\bundle\testdata\sha256sum_correct_4.13.0.txt.sig` and when we use the URI `file:///` the additional `/` is again converted to the root volume so we have double `D:\D:\`

Reply:
This passes on my test Windows VM with:
```
diff --git a/pkg/crc/machine/bundle/metadata_test.go b/pkg/crc/machine/bundle/metadata_test.go
index cff984a98..61134f9b0 100644
--- a/pkg/crc/machine/bundle/metadata_test.go
+++ b/pkg/crc/machine/bundle/metadata_test.go
@@ -209,5 +209,5 @@ func TestVerifiedHash(t *testing.T) {
 func testDataURI(t *testing.T, sha256sum string) string {
        absPath, err := filepath.Abs(filepath.Join("testdata", sha256sum))
        require.NoError(t, err)
-       return fmt.Sprintf("file:///%s", absPath)
+       return fmt.Sprintf("file://%s", filepath.ToSlash(absPath))
 }
```
(`file:///` -> `file://` + use of `filepath.ToSlash()`).
On linux, the test also passes with these changes.

Reply:
ahh, i think i might have used `file:///` instead of `file://`